plugins/by-name/lint/default.nix

```diff
@@ -225,43 +154,57 @@ in
           callback = defaultCallback;
         };
       };
+
+    linters =
+      defaultNullOpts.mkAttrsOf linterType
```

Comment:
Might be nicer to move the let block down here so it's usage is close to its definition.

Reply:
Not needed anymore